simphony/io/h5_cuds.py

```diff
@@ -177,6 +209,28 @@ def get_mesh(self, name):
             raise ValueError(
                 'Mesh \'{n}\` does not exist'.format(n=name))
 
+    def get_lattice(self, name):
+        """Get lattice from file.
+
+        The returned lattice can be used to query
+        and change the related data stored in the file. If the
+        file has been closed then the lattice should
+        no longer be used.
+
+        Parameters
+        ----------
+        name : str
+            name of lattice to return
+        """
+        if name in self._handle.root.lattice:
+            F = self._handle.root.lattice._v_leaves[name].attrs
```

Comment:
I think this is not necessary since the ``__init__`` method of FileLattice will recover the right values.